tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -418,7 +421,7 @@ mod tests {
 
         let streams = CudaStreams::new_multi_gpu();
 
-        let num_blocks = 32;
+        let num_blocks = 16;
```

Comment:
We could have kept 32 blocks to test for the compression/decompression of 64-bit messages?